### PR TITLE
ZeroBrane.Studio version 1.90

### PR DESCRIPTION
**Pull request by sarvex**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103005)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Screenshot (202)
@sarvex Does any installer switches to pass by the installer?

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie. It look like this PR is need to be block.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230420065646

File: manifests/z/ZeroBrane/Studio/1.90/ZeroBrane.Studio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ZeroBrane.Studio
+PackageVersion: 1.90
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://download.zerobrane.com/ZeroBraneStudioEduPack-1.90-win32.exe
+  InstallerSha256: 6649BAC74B39947D1D2E5AD4340F6D116CDABF2580953000FD11BCCEFE71089E
```
Comment (stephengillie) [inline manifests/z/ZeroBrane/Studio/1.90/ZeroBrane.Studio.installer.yaml:10]:
```suggestion
  InstallerSha256: 6649BAC74B39947D1D2E5AD4340F6D116CDABF2580953000FD11BCCEFE71089E
  InstallerSwitches:
    Silent: /s
```

**Comment by BrandonWanHuanSheng** [comment]
@sarvex, It look like suggested change to this PR.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**